### DISCUSSION TITLE
Can we use different suppercell for harmonic and anharmonic fc？

### DISCUSSION THREAD

**Discussion by gangliu-github** [ttadano/alamode#153] [Q&A] [answered]
Dear users and developers,
Can we use different suppercell for harmonic and anharmonic fc？
For example, 3 3 3 suppercell for harmonic fc, 2 2 2 suppercell for 4th fc

Thanks and Best,
Gang Liu

**Comment by ttadano** [comment, marked as answer]
Yes. Please see the description of `FC2XML` in anphon.

**Reply by gangliu-github** [reply]
I get it. Thanks for your reply!
